lib/Ed25519VerificationKey2020.js

```diff
@@ -69,14 +91,24 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
    * @returns {Ed25519VerificationKey2020} - 2020 suite instance.
    */
   static fromEd25519VerificationKey2018({keyPair} = {}) {
+    const publicKeyMultibase = MULTIBASE_BASE58BTC_HEADER +
+      base58btc.encode(new Uint8Array([
+        ...MULTICODEC_ED25519_PUB_HEADER,
+        ...base58btc.decode(keyPair.publicKeyBase58)
+      ]));
     const keyPair2020 = new Ed25519VerificationKey2020({
       id: keyPair.id,
       controller: keyPair.controller,
-      publicKeyMultibase: `z${keyPair.publicKeyBase58}`
+      publicKeyMultibase
     });
 
     if(keyPair.privateKeyBase58) {
-      keyPair2020.privateKeyMultibase = `z${keyPair.privateKeyBase58}`;
+      const privateKeyMultibase = MULTIBASE_BASE58BTC_HEADER +
+        base58btc.encode(new Uint8Array([
+          ...MULTICODEC_ED25519_PRIV_HEADER,
+          ...base58btc.decode(keyPair.privateKeyBase58)
+        ]));
```

Comment:
```suggestion
      const key = base58btc.decode(keyPair.privateKeyBase58);
      const mk = new Uint8Array(
        MULTICODEC_ED25519_PRIV_HEADER.length + key.length);
      mk.set(MULTICODEC_ED25519_PRIV_HEADER);
      mk.set(key, MULTICODEC_ED25519_PRIV_HEADER.length);
      const privateKeyMultibase = MULTIBASE_BASE58BTC_HEADER +
        base58btc.encode(mk);
```

Reply:
Maybe create a quick `_concat` helper seeing as it looks like this repeats at least 4 times in this PR.

Reply:
Fixed in ce7ff1158384b8bf1c65998676bfc5ac3edd604a.